File: src/main/java/com/classroom/misc/MiscellaneousController.java
package com.classroom.misc;

import org.apache.commons.logging.Log;
import org.apache.commons.logging.LogFactory;
import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
public class MiscellaneousController {
    private final Log log = LogFactory.getLog(MiscellaneousController.class);

    @GetMapping("/plans")
    public String welcome(Model model) {
        log.info("received incoming traffic and redirected to plans");
        return "misc/plans";
    }

    @GetMapping("/faq")
    public String faq(Model model) {
        log.info("received incoming traffic and redirected to faq");
        return "misc/faq";
    }

    @GetMapping("/help/support")
    public String support(Model model) {
        log.info("received incoming traffic and redirected to support");
        return "misc/support";
    }
}

File: src/main/kotlin/com/classroom/misc/MiscellaneousController.kt
package com.classroom.misc

import org.apache.commons.logging.LogFactory
import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.GetMapping

@Controller
class MiscellaneousController {
    private val log = LogFactory.getLog(javaClass)

    @GetMapping("/plans")
    fun welcome(): String {
        log.info("received incoming traffic and redirected to plans")
        return "misc/plans"
    }

    @GetMapping("/faq")
    fun faq(): String {
        log.info("received incoming traffic and redirected to faq")
        return "misc/faq"
    }

    @GetMapping("/help/support")
    fun support(): String {
        log.info("received incoming traffic and redirected to support")
        return "misc/support"
    }
}